projects/pgai/db/sql/idempotent/011-vectorizer-int.sql

```diff
@@ -617,6 +822,15 @@ begin
     )
     loop
         raise notice 'Recreating trigger function for vectorizer ID %s', _vec.id;
+        
+        _destination_type := _vec.config->'destination'->>'implementation';
+        if _destination_type = 'table' then
+            _target_schema := _vec.config->'destination'->>'target_schema';
+            _target_table := _vec.config->'destination'->>'target_table';
+        else
+            raise notice 'Skipping vectorizer ID %s because it has a non-table destination', _vec.id;
```

Comment:
wait we have a trigger on destination_column vectorizers so how is this right?

Reply:
This is just the recreation logic, since we don't yet have any destination column vectorizers this wasn't really necessary. But it actually is easy now since the definition just relies  on target table and target schema being null to create the correct trigger definition. So fixed now.